src/pages/basic_data/app_admin/CustomerInstitutionInformationFromCristin.tsx

```diff
@@ -0,0 +1,86 @@
+import { Grid, TextField } from '@mui/material';
+import { useQuery } from '@tanstack/react-query';
+import { useTranslation } from 'react-i18next';
+import { fetchOrganization } from '../../../api/cristinApi';
+import { PageSpinner } from '../../../components/PageSpinner';
+import { Organization } from '../../../types/organization.types';
+import { getUnitTopLevelCode } from '../../../utils/institutions-helpers';
+import { getLanguageString } from '../../../utils/translation-helpers';
+
+interface CustomerInstitutionInformationFromCristinProps {
+  cristinId?: string;
+  organizationData: Organization | null;
+  setName: (string: string) => void;
+  displayName?: string;
+}
+
+export const CustomerInstitutionInformationFromCristin = ({
+  cristinId,
+  organizationData,
+  setName,
+  displayName,
+}: CustomerInstitutionInformationFromCristinProps) => {
+  const { t } = useTranslation();
+  const organizationQuery = useQuery({
+    queryKey: [cristinId],
+    queryFn: cristinId ? () => fetchOrganization(cristinId) : undefined,
+    meta: { errorMessage: t('feedback.error.get_institution') },
+    staleTime: Infinity,
+    cacheTime: 1_800_000, // 30 minutes
+  });
+
+  const customerInformation = organizationData ?? organizationQuery.data;
+  const displayNameFromCristin = getLanguageString(organizationData?.labels, 'nb');
```

Comment:
Prøvde å trigge denne nå ved å hardkode et annet navn (`const displayNameFromCristin = "LOFF"`), og det funker! men får samtidig en warning som jeg ikke finner en åpenbar løsning på:
![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/007e1f04-56fe-4031-a98b-98a80517c017)
Virker som det er tøblete å oppdatere Formik state fra andre komponentetr, uten at jeg ser helt hvorfor. En liten side-ting er jo at det ville trigge en ekstra re-render da, så kan være at det er en warning for å hjelpe å unngå inifite rerenders som oft kommer når man oppdatere (formik) state i selve render funksjonen 🤔 skal grave litt mer

Reply:
Tror jeg fant en løsning på det:
```
const organization = organizationQuery.data;
useEffect(() => {
  const displayNameFromCristin = getLanguageString(organization?.labels, 'nb');
  if (displayName !== displayNameFromCristin && displayNameFromCristin.length > 0) {
    setName(displayNameFromCristin);
  }
}, [setName, displayName, organization?.labels]);
```

- Ta vekk `organizationData`, da det nå er dobbeltlagring av org-data fra cristin, og trigger unødige rerender(s) når den oppdateres i `CustomerInstitutionMetadataForm.tsx`
- Legg (formik) state updates i en `useEffect`, slik at den bare initieres når den har endrede dependencies

Sjelden et godt tegn når man tar i bruk `useEffect` for tiden, men tror det er det beste jeg finner her :/

Reply:
Siden vi ikke skal vise frem displayname noen som helst plasser bortsett fra i customer-lista, så fjernet jeg den. 

Vi bør kanskje hente inn displayName til lista ved å fetche disse dataene, som du sier er ting cachet, så bør ikke være så ille.